pkg/virt-controller/watch/migration-backoff_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package watch
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	virtv1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("Migration backoff", func() {
```

Comment:
Any chance we could unit test the GC, by creating and abandoning a bunch of entries or something? That `infoMap` makes me nervous...

Reply:
As it's implemented right now not really, I'll think of something, even if I'm not sure there is a real need as GC() itself is already unit tested (in its own unit tests).

Reply:
Amazing thank you!
/approve